### ISSUE TITLE
What is the difference between motion_name_clip_x and motion_name motions in the AIST dataset?

### ISSUE THREAD

**Issue by sohananisetty** [IDEA-Research/Motion-X#13]
For example, there is Dance_Break_3_Step_clip_[1-19].npy and Dance_Break_3_Step.npy? I would assume Dance_Break_3_Step is a concatenation of all individual clips but it the motion length is very small.

**Comment by linjing7** [comment]
Hi, thanks for your interest! In AIST, each dance category (e.g., Dance_Break_3_Step) would have several videos and motion sequences. Thus, we add clip_[1-N] to distinguish these motion sequences.

**Comment by sohananisetty** [comment]
Yeah that part is clear. What does Dance_Break_3_Step with no clip number represent?

**Comment by linjing7** [comment]
It denotes `Dance_Break_3_Step_clip_0`.

**Comment by sohananisetty** [comment]
Got it. Thanks! Also, can you share the mapping between names used in your AIST dataset and the official names?
